edx_sga/utils.py

```diff
@@ -9,10 +9,33 @@
 
 import pytz
 from django.conf import settings
-from django.core.files.storage import default_storage
+from django.core.files.storage import default_storage as django_default_storage, get_storage_class
 from edx_sga.constants import BLOCK_SIZE
 
 
+def get_default_storage():
+    """
+    Get config for storage from settings, use DJango's default_storage if no such settings defined
+    """
+    # .. setting_name: SGA_STORAGE_SETTINGS
+    # .. setting_default: {}
+    # .. setting_description: Specifies the storage class and keyword arguments to use in the constructor
```

Comment:
I think this might be a better description.

```suggestion
    # .. setting_description: Specifies the storage class and keyword arguments to use in the constructor. 
    # .. Default Storage would be used if this setting is not specified.
```

Reply:
Done